### ISSUE TITLE
Run the final evaluation using the full training data?

### ISSUE THREAD

**Issue by GMN23362** [KMnP/vpt#56]
Thanks for your great work. I'm wondering about why can test on training data?
![image](https://github.com/KMnP/vpt/assets/74642115/a2e91ce7-9db9-40f8-a831-7db29074ea15)
As far as I known, generally, we use dataset seperated from training data as test data. I'm quite puzzled. I'll be so glad to see your answer~

**Comment by Tsingularity** [comment]
thanks for your interest in our work! I think there's some mis-understanding here. We use the 800-200 split on the training set to grid-search hyper-parameters and select the best one. Then use it to run another finetuning on the whole 1000 training set to get the final model. The evaluation is on the test set, which has never been used in training. Hope this helps!
